_site/index.html

```diff
@@ -547,6 +588,36 @@ <h3>T-Z</h3>
           </div>
         </section>
 
+        <section>
+          <p class="word__title" id="non-binary">
+            Non-Binary
+            <span class="word__speech">
+              adj
+            </span>
+          </p>
+          
+          <p class="word__definition">
+            someone whose gender does not fit into the male-female gender binary.<br><br>
+            
+            For example, a non-binary person may have some gender between male and female, have multiple genders, no gender, a culturally-specific gender (e.g. two-spirit).
```

Comment:
For example, a non-binary person may have some gender between male and female, have multiple genders, no gender, a culturally-specific gender (e.g., two-spirit).